Assignment/solution_student/css/style.css

```diff
@@ -46,4 +55,134 @@ body {
 
 /******************************************
 /* SECTIONS
-/*******************************************/
\ No newline at end of file
+/*******************************************/
+
+header {
+	background-color: #033048;
+}
+
+header h1 {
+	float: left;
+	color: white;
+	padding-left: 100px; 
+	padding-bottom: 50px; 
+}
+header nav {
+	float: right;
+	font-size: 24px; 
+	margin-top: 25px;
+	margin-right: 40px;
+}
+
+header a {
+	text-decoration: none;
+	color: #f9e42e;
+	padding: 10px; 
+	font-size: 24px; 
+}
+
+.clearfix:after {
+	clear: both;
+	padding-right: 300px; 
+	padding-left: 400px; 
+	width: 80%; 
+}
+
+#articles {
+	float: left;
+	width: 70%; 
+	padding-left: 100px; 
+	padding-right: 50px;
+	padding-bottom: 20px; 
+}
+
+#articles img {
+	width: 70%; 
+}
+
+#articles h1 {
+	width: 70%; 
+}
+
+.side-menu {
+	float: right;
+	width: 20%; 
+	padding-right: 300px; 
+	margin-right: 100px; 
+}
+
+.side-menu a {
+	text-decoration: none;
+}
+
+.side-menu img {
+	padding-right: 50px; 
+}
+
+.side-menu h2 {
+	font-size: 32px; 
+	font-family: 'Open sans' , 'extra bold' ;
```

Comment:
Okay, I see what you're saying about the font weights — so you were trying to declare a font-weight on the font-family property, which is specifically just to declare the font-family, not font-weight. If you want to change the font-weight then you can with the `font-weight` property, and when using these number values that they provide (that's the part in the code you included in the header: `family=Open+Sans:300,300italic,400,600,800` — so `extra bold` would be `font-weight: 800;`, semibold: `600`, and light: `300`. Make sense?